### PR TITLE
[tensorflow] Bug fixes to TF2.8 E3 images

### PR DESCRIPTION
**Pull request by arjkesh**
*GitHub Issue #, if available:*

Note: If merging this PR should also close the associated Issue, please also add that Issue # to the Linked Issues section on the right. 

### Description
- Update license file
- Correct env variables

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: tensorflow/inference/docker/2.8/py3/cu112/Dockerfile.gpu [hunk 1]
```diff
@@ -74,6 +74,8 @@ RUN apt-get update \
     tk-dev \
     libgdbm-dev \
     libc6-dev \
+    # remove libsasl2-2 after Nvidia docker upstream fix CVE-2022-24407
+    libsasl2-2 \
```
Comment (shantanutrip) [inline tensorflow/inference/docker/2.8/py3/cu112/Dockerfile.gpu:78]: Can we use the `-only-upgrade` solution as done in PR1685. It works for e3 containers as well?

File: tensorflow/training/docker/2.8/py3/example/Dockerfile.gpu [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Expecting base image to be the image built by ../cu112/Dockerfile.e3.gpu
+# Expecting base image to be the image built by ../cu112/Dockerfile.gpu e3 target
```
Comment (shantanutrip) [inline tensorflow/training/docker/2.8/py3/example/Dockerfile.gpu:1]: Just wanted to confirm this. E3 images act as the base for example images?

Comment (arjkesh) [reply tensorflow/training/docker/2.8/py3/example/Dockerfile.gpu:1]: yes, e3 gpu images are the base for example images

**Review by shantanutrip — APPROVED** [review]
LGTM

**Pull request merged**